src/tooling_webserver.nim

```diff
@@ -40,12 +41,12 @@ router routes:
     let response = %*
         {
             "exit_status": exit_status,
-            "result": nil
+            "results": %*{}
```

Comment:
We made this a JObject to allow key/value pairs to be added later on for the results file paths to results file paths contents mapping.